hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -188,6 +191,14 @@ public String getOwnerName() {
     return ownerName;
   }
 
+  public Map<Integer, List<OmKeyLocationInfo>> getCurrentlocationsPartsMap() {
+    return currentlocationsPartsMap;
+  }
+
+  public Map<Integer, Long> getCurrentDataSizePartsMap() {
+    return currentDataSizePartsMap;
+  }
+
```

Comment:
Do we need this? It's not persisted right?

Reply:
Removed.